Description: To find gzstream.h during Debian package building.
Author: Charles Plessy <plessy@debian.org>
Forwarded: no
diff --git a/src/utils/gzstream/gzstream.C b/src/utils/gzstream/gzstream.C
index 8cb4590..bbb4ba8 100644
--- a/src/utils/gzstream/gzstream.C
+++ b/src/utils/gzstream/gzstream.C
@@ -26,7 +26,7 @@
 // Standard C++ Library".
 // ============================================================================
 
-#include <gzstream.h>
+#include "gzstream.h"
 #include <iostream>
 #include <string.h>  // for memcpy
 
